### ISSUE TITLE
kubeconfig without a current-context crashes telepresence.

### ISSUE THREAD

**Issue by philips** [datawire/telepresence#164]
### What were you trying to do?

telepresence -m inject-tcp -n test --run-shell

### What did you expect to happen?

Working shell

### What happened instead?

Traceback.

I am using a Tectonic cluster and the kubeconfig doesn't include a cluster-context entry in the yaml. By adding the following to the bottom of my kubeconfig this bug went away.

```
  name: local
current-context: local
```

### Automatically included information

Command line: `['/usr/local/bin/telepresence', '-m', 'inject-tcp', '-n', 'test', '--run-shell']`
Version: `0.49`
Python version: `3.6.1 (default, Apr  7 2017, 18:53:53) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]`
kubectl version: `Client Version: v1.7.0-alpha.4`
oc version: `(error: [Errno 2] No such file or directory: 'oc')`
OS: `Darwin Brandons-MacBook-Pro.local 16.6.0 Darwin Kernel Version 16.6.0: Fri Apr 14 16:21:16 PDT 2017; root:xnu-3789.60.24~6/RELEASE_X86_64 x86_64`
Traceback:

```
Traceback (most recent call last):
  File "/usr/local/bin/telepresence", line 95, in call_f
    return f(*args, **kwargs)
  File "/usr/local/bin/telepresence", line 1452, in go
    runner, remote_info, args, env, subprocesses, socks_port, ssh_port
  File "/usr/local/bin/telepresence", line 1210, in run_local_command
    runner.kubectl_cmd, "config", "current-context"
  File "/usr/local/bin/telepresence", line 198, in get_output
    **kwargs).strip(), "utf-8"
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['kubectl', 'config', 'current-context']' returned non-zero exit status 1.

```

Logs:

```
rictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', 'telepresence@localhost', '/bin/true'],)... Running: (['ssh', '-q', '-p', '52928', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', 'telepresence@localhost', '/bin/true'],)... Forwarding from 127.0.0.1:52928 -> 8022
Forwarding from [::1]:52928 -> 8022
Running: (['ssh', '-q', '-p', '52928', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', 'telepresence@localhost', '/bin/true'],)... Handling connection for 52928
 ran!
Running: (['ssh', '-q', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', '-oServerAliveInterval=1', '-oServerAliveCountMax=10', '-N', '-p', '52928', 'telepresence@localhost', '-L', '127.0.0.1:52942:127.0.0.1:9050'],)...Running: (['kubectl', '--context', '', '--namespace', 'default', 'exec', 'test-3321722284-0h828', '--container', 'test', 'env'],)...Handling connection for 52928
 ran!
Running: (['kubectl', 'config', 'current-context'],)...error: current-context is not set

```

**Comment by itamarst** [comment]
Thanks for the bug report! I'll try to get this fixed.

**Comment by itamarst** [comment]
@philips I just released v0.50 with fixes for your two bugs. Let me know if you have any other problems!

What's your use case for Telepresence, by the way?

**Comment by philips** [comment]
Just trying it out, neat project, good work!
